Unable to develop locally or deploy with a different repo using the same markdown file and pmtiles dataset

When creating a new repository, the following leads to `source invalid` errors after copying the example markdown file and data dependency from this repository:

* github.com -> create new repository
* Then follow https://observablehq.com/framework/getting-started:
* Run `npm init @observablehq`(select "include sample files" and "npm" and "initialize git repository")
* Copy files from this repo into the new one:
```
git clone git@github.com:bdon/observable-framework-maps.git
# copy markdown file
cp observable-framework-maps/docs/example-map.md observable-framework-maps-replica/docs
# copy markdown file data dependency
cp observable-framework-maps/docs/data/cb_2018_us_zcta510_500k_nolimit.pmtiles observable-framework-maps-replica/docs/data
```
* install the CORS Unblock browser extension (related to https://github.com/bdon/observable-framework-maps/issues/1): https://chromewebstore.google.com/detail/cors-unblock/lfhmikememgdcahcdlaciloancbhjino
* enter the folder and start the development server:
```
cd observable-framework-maps-replica
npm run dev
```

This yields the following error:
```
Error: Invalid source undefined
```

With the JavaScript console error log:
```
socket open
client.js:440 ↑ Object
client.js:133 Error: Invalid source undefined
    at e (_esm.js:7:81805)
    at Ae (_esm.js:7:81995)
    at new e (_esm.js:7:91750)
    at Module.ii (_esm.js:7:95048)
    at body (example-map:78:24)
    at define (runtime.js:1318:33)
rejected @ client.js:133
favicon.ico:1 
        
        
       Failed to load resource: the server responded with a status of 404 (Not Found)
undefined:1 
        
        
       Failed to load resource: the server responded with a status of 404 (Not Found)
_esm.js:7 Error: Bad response code: 404
    at oe.<anonymous> (_esm.js:7:13731)
    at Generator.next (<anonymous>)
    at s (_esm.js:7:66)
```

I tried the following to debug:
- [x] checked that I can deploy this repo successfully: https://jaanli.github.io/observable-framework-maps/example-map (from this fork: https://github.com/jaanli/observable-framework-maps/)
- [x] checked that I can do local development on this repo or the fork 

I might be missing something basic as it's my first time doing something like this.

`package.json` from this: 
```
{
  "type": "module",
  "private": true,
  "scripts": {
    "clean": "rimraf docs/.observablehq/cache",
    "build": "rimraf dist && observable build",
    "dev": "observable preview",
    "deploy": "observable deploy",
    "observable": "observable"
  },
  "dependencies": {
    "@observablehq/framework": "latest",
    "d3-dsv": "^3.0.1",
    "d3-time-format": "^4.1.0"
  },
  "devDependencies": {
    "rimraf": "^5.0.5"
  },
  "engines": {
    "node": ">=18"
  }
}
```

--- a/docs/example-map.md
+++ b/docs/example-map.md
@@ -71,7 +71,7 @@ div.style = "height: 400px;";
 const map = L.map(div).setView(center,zoom);
 var baseLayer = protomapsL.leafletLayer({url:'https://api.protomaps.com/tiles/v3/{z}/{x}/{y}.mvt?key=1003762824b9687f', theme:'black'})
 baseLayer.addTo(map);
-var layer = protomapsL.leafletLayer({url:areas._url, maxDataZoom:7, paintRules: [
+var layer = protomapsL.leafletLayer({url:areas.href, maxDataZoom:7, paintRules: [
   {
       dataLayer: "zcta",
       symbolizer: new protomapsL.PolygonSymbolizer({
@@ -115,7 +115,7 @@ const map = new maplibregl.Map({
 map.on("load", () => {
   map.addSource("zcta", {
       type: "vector",
-      url: `pmtiles://${areas._url}`
+      url: `pmtiles://${areas.href}`
   })
 
   map.addLayer({

